.env.sample

```
@@ -0,0 +1,12 @@
+DATABASE_URL = "mysql://password@ipAddress/sql"
+PORT = 3000
+
+TYPEORM_CONNECTION = mysql
```

```suggestion
TYPEORM_CONNECTION = . . .
```
마찬가지로 `mysql`이라고 직접 명시가 아닌 `rdbms` 등으로 표현해주세요.